src/main/java/com/softserve/academy/spaced/repetition/service/CardService.java

```diff
@@ -11,29 +11,104 @@
 import java.io.OutputStream;
 import java.util.List;
 
+/**
+ * This interface works with cards.
+ */
 public interface CardService {
+    /**
+     * Returns cards that belong deck with the given identifier depends on learning regime.
+     *
+     * @param deckId must not be {@literal null}.
+     * @return all cards with the given deck's id (by default 10)
+     * @throws NotAuthorisedUserException
+     */
     List<Card> getLearningCards(Long deckId) throws NotAuthorisedUserException;
 
+    /**
+     * Retrieves an card by its id.
+     *
+     * @param id must not be {@literal null}.
+     * @return the card with the given id or {@literal null} if none found.
+     */
     Card getCard(Long id);
 
+    /**
+     * Adds card to deck with the given identifier.
+     *
+     * @param card   must not be {null}.
```

Comment:
Maybe need to add more description for params.
We can discuss it.

Reply:
ok